### PR TITLE
Merge bitcoin#13793, #14208

### PR DESCRIPTION
**Pull request by 5tefan**

### CODE REVIEW

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
pls see https://github.com/UdjinM6/dash/commits/pr4328

**Comment by UdjinM6** [comment]
pls squash the fix (877f254) into the corresponding backport commit (6854734)

File: src/dummywallet.cpp [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #include <util/system.h>
 #include <walletinitinterface.h>
 
-#include <stdio.h>
+class CWallet;
```
Comment (UdjinM6) [inline src/dummywallet.cpp:9]: 14208:
```suggestion
#include <stdio.h>

class CWallet;
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: src/dummywallet.cpp (hunk 1 above)
Comment (5tefan) [reply src/dummywallet.cpp:11]: Done + squashed into 1cea09e.

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**